### PR TITLE
Update Campground Details Page

### PR DESCRIPTION
**Pull request by swill93**
issue #22 / Sprint 2 PBI 4, Task 2

1. Modified HTML in ViewCamp.js to have separate sections to organize page
2. Added a "Leave a Review" section to the page
3. Added a Similar Campgrounds section to the page
4. Added DetailsPage.css to style the page to align more with the wireframe design

### CODE REVIEW

**Review by wildal99 — APPROVED** [review]
Everything Looks Good

**Review by aeSzelag — APPROVED** [review]
Everything looks like its good to me.

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   },
   "homepage": "https://github.com/COSC481W-2024Fall/CampQuest#readme",
   "dependencies": {
+
```
Comment (sjiang1) [inline package.json:26]: remove the extra whitespace

**Comment by sjiang1** [comment]
Grading Feedback:

Quality of Code: Good
Participation in Code Review: Good
Post-Review Changes: Didn't change anything after the review.

Deductions: 
(-2 points): Some commit messages do not adhere to the required format specified in the lecture.
(-5 points): No evidence of testing is provided in the pull request. If testing is documented in another pull request, please inform me so that the grade can be updated accordingly.

**Pull request merged**